samples/test/cluster.js

```diff
@@ -75,6 +77,22 @@ describe.skip('Cluster Snippets', () => {
     clusterSnippets.setMeta(INSTANCE_ID, CLUSTER_ID);
   });
 
+  it('should create a backup of a table', () => {
+    clusterSnippets.createBackup(INSTANCE_ID, CLUSTER_ID, TABLE_ID, BACKUP_ID);
```

Comment:
Are these all async functions?  Do we need any assertions to make sure they ran successfully?  Is there a chance these tests are all skipped in the describe block?